accelerometer/summariseEpoch.py

```diff
@@ -218,11 +219,14 @@ def check_daylight_savings_crossover(e, startTime, endTime, summary,
         for t in localTime._utc_transition_times:
             if t>startTime:
                 transition = t
+                transition = transition - pd.DateOffset(seconds = 1)
+                transition = utc_tz.localize(transition)
+                transition = localTime.normalize(transition)
+                transition = transition.replace(tzinfo = None)
+                transition = transition + pd.DateOffset(seconds = 1)
                 break
         # If Autumn crossover time, adjust transition time plus remove 1hr chunk
         if offset == -1:
-            # pytz stores dst crossover at 1am, but clocks change at 2am local
-            transition = transition + pd.DateOffset(hours=1)
```

Comment:
since the previous steps have already resulted in correct output (2am) for both spring and fall crossover, this step is no longer necessary